.github/workflows/test.yml

```diff
@@ -0,0 +1,58 @@
+name: test
+
+on: [push, pull_request]
+
+jobs:
+  ansible-molecule:
+    runs-on: macos-10.15
```

Comment:
Why MacOS?

Reply:
Currently Vagrant support for nested virtualization works only on GitHub Action MacOS environment:
https://github.com/actions/virtual-environments/issues/433

Reply:
How weird